package.json

```diff
@@ -63,6 +63,7 @@
     "idb": "^4.0.3",
     "json-stable-stringify": "^1.0.1",
     "jsqr": "^1.2.0",
+    "jss": "^10.0.0",
```

Comment:
Why add jss as a explicit dependency? It should update with @material-ui

Reply:
Because everything require'd should be resolvable at the top level. We should not assume that the package manager puts everything flat at node_modules. In fact, anything not listed but required *should* be treated as some sort of deplist escape.

https://medium.com/pnpm/flat-node-modules-is-not-the-only-way-d2e40f7296a3

Reply:
Because everything require'd should be resolvable at the top level. We should not assume that the package manager puts everything flat at node_modules. In fact, anything not listed but required *should* be treated as some sort of deplist escape.

https://medium.com/pnpm/flat-node-modules-is-not-the-only-way-d2e40f7296a3